Documentation/nfs.md

```diff
@@ -124,7 +175,6 @@ metadata:
   name: rook-nfs
   namespace: rook-nfs
 spec:
-  serviceAccountName: rook-nfs
```

Comment:
will this example work for both cases where the user is on OpenShift and on vanilla upstream k8s?